pkg/virt-operator/resource/generate/components/secrets.go

```diff
@@ -251,7 +251,7 @@ func NewCertSecrets(installNamespace string, operatorNamespace string) []*k8sv1.
 // nextRotationDeadline returns a value for the threshold at which the
 // current certificate should be rotated, 80% of the expiration of the
```

Comment:
Is the `80%` figure still relevant?

Reply:
It's technically still the default, but agree that saying that in the function comment is confusing/not helpful..